lib/index.js

```diff
@@ -37,6 +37,7 @@ function NwBuilder(options) {
         macIcns: false,
         macZip: false,
         macPlist: false,
+        mergeZip: true,
```

Comment:
Isn't zipping on those other platforms defaulted to true? Right now, there is no option for Win/Linux, so it always zips. Hence: a default true value.